### PR TITLE
Tool 989/force code sign properties

### PR DESCRIPTION
**Pull request by BirmacherAkos**
Test fix in the other PR:
https://github.com/bitrise-io/xcode-project/pull/36

---

We need to make sure the given ios project’s codessigning settings are matching to the auto prov step generated codesigning files, so we need to force the project’s:

- codesigning style 
- codesign identity
- provisioning profile

team id

```
target_attributes['ProvisioningStyle'] = 'Manual'
target_attributes['DevelopmentTeam'] = development_team
target_attributes['DevelopmentTeamName'] = ''

build_settings = build_configuration.build_settings
codesign_settings = {
  'CODE_SIGN_STYLE' => 'Manual',
  'DEVELOPMENT_TEAM' => development_team,

  'CODE_SIGN_IDENTITY' => code_sign_identity,
  'CODE_SIGN_IDENTITY[sdk=iphoneos*]' => code_sign_identity,

  'PROVISIONING_PROFILE_SPECIFIER' => '',
  'PROVISIONING_PROFILE' => provisioning_profile_uuid,
  'PROVISIONING_PROFILE[sdk=iphoneos*]' => provisioning_profile_uuid
}
```

### CODE REVIEW

File: xcodeproj/xcodeproj.go [hunk 1]
```diff
@@ -212,59 +218,166 @@ func Open(pth string) (XcodeProj, error) {
 		return XcodeProj{}, err
 	}
 
-	objects, projectID, err := open(pth)
+	raw, objects, projectID, format, err := open(pth)
```
Comment (godrei) [inline xcodeproj/xcodeproj.go]: i think the raw representation and the format is strictly connected, i would suggest to change the output's order: `format, raw, objects, projectID, err`

File: xcodeproj/configuration_list.go [hunk 2]
```diff
@@ -36,8 +40,55 @@ func parseConfigurationList(id string, objects serialized.Object) (Configuration
...
+	object, err := objects.Object(targetID)
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch target buildConfigurationList, the project objects with ID (%s) is not found, error: %s", p.Proj.ID, err)
+	}
+	buildConfigurationListID, err := object.String("buildConfigurationList")
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch target's (%s) buildConfigurationList, error: %s", targetID, err)
+	}
+
+	buildConfigurationList, err := objects.Object(buildConfigurationListID)
```
Comment (trapacska) [inline xcodeproj/configuration_list.go]: You can return here

File: xcodeproj/xcodeproj.go (hunk 1 above)
Comment (trapacska) [reply xcodeproj/xcodeproj.go]: NOTE level: I agree.

File: xcodeproj/configuration_list.go (hunk 2 above)
Comment (BirmacherAkos) [reply xcodeproj/configuration_list.go]: Fixed

File: xcodeproj/xcodeproj.go (hunk 1 above)
Comment (BirmacherAkos) [reply xcodeproj/xcodeproj.go]: Fixed

**Pull request merged**